_tour/basics.md

```diff
@@ -74,11 +71,11 @@ Variables are like values, except you can re-assign them. You can define a varia
 
 ```tut
 var x = 1 + 1
-x = 3 // This compiles because "x" is declared with the "var" keyword.
+x = 3 // This compiles because "x" is declared with the "var" keyword:
```

Comment:
This change is incorrect!  The comment refers to _this_ line, not the _next_ line which is what the colon suggests!  No colon--the original was correct.